master/docs/relnotes/0.9.0b2.rst

```diff
@@ -0,0 +1,89 @@
+Release Notes for Buildbot 0.9.0b2
+==================================
+
+..
+    Any change that adds a feature or fixes a bug should have an entry here.
+    Most simply need an additional bulleted list item, but more significant
+    changes can be given a subsection of their own.
+
+The following are the release notes for Buildbot 0.9.0b2.
+Buildbot 0.9.0b2 was released on the 02nd of August, 2015.
+
+Master
+------
+
+Features
+~~~~~~~~
+
+* Mercurial hook was updated and modernized.
+  It is no longer necessary to fork.
+  You can now extend PYTHONPATH via the hook config which allows you to use a buildbot-venv instead of installing buildbot to the system site-packages.
+  Added documentation inside the hook.
+  Misc. clean-up and reorganization in order to make the code a bit more readable.
+
+
+* UI templates can now be customizable.
+  You can provide html or jade overrides to the www plugins, end then customize your look and feel.
+
+
+Fixes
+~~~~~
+
+* Fix setup for missing www.hooks module
+
+* Fix setup to install only on recent version of pip.
+  This prevents unexpected upgrade to nine from people who just use ``pip install -U buildbot``
+
+* Fix a crash in the git hook.
+
+* add checks to enforce slavenames are identifiers.
+
+
+Deprecations, Removals, and Non-Compatible Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changes for Developers
+~~~~~~~~~~~~~~~~~~~~~~
+
+* The :py:class:`~buildbot.config.BuilderConfig` ``nextSlave`` keyword argument takes a callable.
+  This callable now receives :py:class:`~buildbot.process.buildrequest.BuildRequest` instance in its signature as 3rd parameter.
+  **For retro-compatibility, all callable taking only 2 parameters will still work**.
+
+* Data api provides a way to query the build list per slave.
+
+* Data api provides a way to query some build properties in a build list.
+
+Slave
+-----
+
+* buildbot-slave now requires python 2.6
+
+Features
+~~~~~~~~
+
+* Schedulers: the ``codebases`` parameter can now be specified in a simple list-of-strings form.
+
+Fixes
+~~~~~
+
+* Fix a race condition in the integration tests
```

Comment:
As we haves fixes above
* A race conditon in the integrations tests (actually two were fixed :))